src/web/components/ClickToView.tsx

```diff
@@ -89,74 +113,115 @@ const Headline = ({
 
 const Body = ({
 	children,
-	width,
+	role,
 }: {
 	children: React.ReactNode;
-	width: number;
-}) => (
-	<div
-		className={css`
-			${width > 300 ? textSans.small() : textSans.xsmall()}
-			a {
-				${width > 300 ? textSans.small() : textSans.xsmall()}
-			}
-		`}
-	>
-		{children}
-	</div>
-);
+	role: RoleType;
+}) => {
+	const textSize = roleTextSize(role);
+	return (
+		<div
+			className={css`
+				${textSize}
+				a {
+					${textSize}
+				}
+			`}
+		>
+			{children}
+		</div>
+	);
+};
+
+const AcceptButton = ({
+	role,
+	handleClick,
+}: {
+	role: RoleType;
+	handleClick: Function;
+}) => {
+	return (
+		<div>
```

Comment:
Do we need this div?

Reply:
Yes, otherwise it seems the button expands to be the full width of the component.